tests/commands/base/test_verify_requires_python.py

```diff
@@ -0,0 +1,51 @@
+import sys
+
+import pytest
+
+from briefcase.config import GlobalConfig
+from briefcase.exceptions import BriefcaseConfigError, UnsupportedPythonVersion
+
+
+def _get_global_config(requires_python):
+    return GlobalConfig(
+        project_name="pep621-requires-python-testing",
+        version="0.0.1",
+        bundle="com.example",
+        requires_python=requires_python,
+    )
+
+
+def test_no_requires_python(base_command, my_app):
+    """If requires-python isn't set, no verification is necessary."""
+
+    base_command.global_config = _get_global_config(requires_python=None)
+    base_command.verify_required_python(my_app)
+
+
+def test_requires_python_met(base_command, my_app):
+    """Validation passes if requires-python specifies a version lower than the running interpreter."""
+
+    major, minor, micro, *_ = sys.version_info
```

Comment:
This is a good test; but it would be better if it were extended with `pytest.mark.parameterize` to verify a range of specifiers that meet requirements (>, >=, <, <=, ==), rather than a single "lower than" test.

Reply:
:+1: Done in https://github.com/beeware/briefcase/pull/2029/commits/dde2dafcc01646aceab4475d4e4718d9c539fa76